src/vanilla/DanModel/DanCodeGenerator.cs

```diff
@@ -35,18 +39,64 @@ public static class DanCodeGenerator
         private const string azureProxyImport = "com.microsoft.azure.v2." + azureProxyType;
         private const string azureProxyType = "AzureProxy";
 
-        public static JavaFile GetAzureServiceManagerJavaFile(CodeModelJva codeModel, Settings settings)
+        private const string implPackage = "implementation";
+        internal const string modelsPackage = ".models";
+
+        public static readonly IDictionary<KeyValuePair<string, string>, string> pageClasses = new Dictionary<KeyValuePair<string, string>, string>();
```

Comment:
I would be happier with this kind of stuff being instance variables and a bunch of these methods becoming instance methods in order to access them, but until (if ever?) AutoRest allows a generator to run as a daemon consuming multiple Swaggers in parallel, it's not essential.